tox.ini

```diff
@@ -42,7 +42,8 @@ deps =
     ci_watson
     xdist: pytest-xdist
     cov: pytest-cov
-    devdeps: git+https://github.com/astropy/astropy
+    devdeps: astropy>=0.0.dev0
+    devdeps: numpy>=0.0.dev0
```

Comment:
This isn't quite right, see https://github.com/spacetelescope/romancal/pull/695#discussion_r1189076034 and https://github.com/spacetelescope/romancal/pull/695#discussion_r1189078751 for details.